README.md

```diff
@@ -993,6 +1005,19 @@ RNCallKeep.addEventListener('checkReachability', () => {
 
 ```
 
+### onHasActiveCall
+
+_Android only._
+
+A listener to tells the JS side if a native call has been answered while there is active self managed call
```

Comment:
"A listener that tells the JS side if a native call has been answered while there was an active self managed call" could be a more correct description imo